src/components/Data/Indexes/Page.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <b-container class="IndexesPage">
+    <headline>
+      Indexes
+      <b-button
+        class="float-right mt-3"
+        data-cy="IndexesPage-createBtn"
+        v-if="canCreateIndex()"
+        variant="primary"
+        :title="
+          !canCreateIndex() ? `Your rights disallow you to create indexes` : ''
+        "
+        @click.prevent="openCreateModal"
+      >
+        <i class="fa fa-plus-circle left" />
+        Create an index
+      </b-button>
+    </headline>
+
+    <list-not-allowed v-if="!canSearchIndex()" />
+    <template v-else>
+      <b-row class="mb-3">
+        <b-col sm="8" class="text-secondary">
+          {{ tableItems.length }}
+          {{ tableItems.length === 1 ? 'index' : 'indexes' }}
+        </b-col>
+        <b-col sm="4">
+          <b-input-group>
+            <template v-slot:prepend>
+              <b-input-group-text>Filter</b-input-group-text>
+            </template>
+            <b-form-input v-model="filter"></b-form-input>
+          </b-input-group>
+        </b-col>
+      </b-row>
+
+      <b-table
+        striped
+        outlined
+        show-empty
+        sticky-header="1000px"
+        :items="tableItems"
+        :fields="tableFields"
+        :filter="filter"
+      >
+        <template v-slot:empty>
+          <h4 class="text-secondary text-center">There is no index.</h4>
+          <p class="text-secondary text-center" v-if="canCreateIndex()">
+            You can create one by hitting the button above.
+          </p>
+        </template>
+        <template v-slot:emptyfiltered>
+          <h4 class="text-secondary text-center">
+            There is no index matching your filter.
+          </h4>
+        </template>
+        <template v-slot:cell(icon)>
+          <i class="fa fa-2x fa-database mr-2"></i>
+        </template>
+        <template v-slot:cell(indexName)="indexName">
+          <router-link
+            :data-cy="`IndexesPage-name--${indexName.value}`"
+            :to="{
+              name: 'DataIndexSummary',
+              params: { index: indexName.value }
+            }"
+          >
+            {{ indexName.value }}
```

Comment:
![image](https://user-images.githubusercontent.com/3192870/73256208-cf432880-41c1-11ea-941c-f010b4429955.png)

plus, In my opinion, it's not easy to know that you can click on the name of the index to see its contents without passing the cursor over it.